### PR TITLE
add category ltree

### PR DESCRIPTION
**Pull request by MagicTearsAsunder**

### CODE REVIEW

File: annotation/app/schemas/categories.py [hunk 1]
```diff
@@ -43,7 +43,6 @@ class Config:
 
 
 class CategoryResponseSchema(CategoryInputSchema):
-    # To be removed
     parents: Optional[List[dict]] = Field(default=[])
```
Comment (khyurri) [inline annotation/app/schemas/categories.py:46]: `parents` and `children` are not Optional, they cannot be None in response, always list

**Comment by cakeinsauce** [comment]
Just want to mention here about this.
https://github.com/epam/badgerdoc/blob/main/CONTRIBUTING.md#commit-messages

File: annotation/app/categories/services.py [hunk 2]
```diff
@@ -190,6 +190,20 @@ def filter_category_db(
...
+) -> None:
+    tree = category_db.tree
+    nlevel = len(tree) - 1
+    query = db.query(Category).filter(Category.tree.op('<@')(tree))
+
+    new_path = func.subpath(Category.tree, nlevel)
+    if new_parent:
+        new_path = new_parent.tree.path + new_path
+
+    query.update(values={'tree': new_path}, synchronize_session=False)
```
Comment (MagicTearsAsunder) [inline annotation/app/categories/services.py:204]: please check if `synchronize_session` arg has a proper value:
[https://docs.sqlalchemy.org/en/14/orm/session_basics.html#selecting-a-synchronization-strategy](url)

**Comment by khyurri** [comment]
Hi, @MagicTearsAsunder 
Is this PR ready to be merged? If so, could you please fix conflicts?

**Comment by MagicTearsAsunder** [comment]
> Hi, @MagicTearsAsunder 
> 
> Is this PR ready to be merged? If so, could you please fix conflicts? 
> 
> 

Conflicts?

**Pull request merged**